<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mpnTesting</class>
 <widget class="QWidget" name="mpnTesting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MPN检测</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pb_FEC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>210</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FEC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>耐热大肠菌群</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pb_TOL">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>210</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_TOL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>总大肠菌群</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_back">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1至-2次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-1至-3次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-2至-4次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-3至-5次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-4至-6次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-5至-7次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-6至-8次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-7至-9次方</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-8至-10次方</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lb_FEC_info">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测试梯度：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
